<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVesselExtractor</class>
 <widget class="QDialog" name="DialogVesselExtractor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选取CT数据源</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>原始CT：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="nonContrastCTComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>血管增强CT：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="enhanceCTComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="confirmBtn">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
